tox.ini

```diff
@@ -66,4 +66,4 @@ extras =
     doc
     cesium
 commands =
-    sphinx-build -d "{toxworkdir}/docs_doctree" docs/source "{toxworkdir}/docs_out" --color -vW -bhtml
+    sphinx-build -d "{toxworkdir}/docs_doctree" docs/source "{toxworkdir}/docs_out" --color -v -bhtml
```

Comment:
Why are we no longer having this flag?

Reply:
I'm getting tons of warnings locally 😓 In fact, [the logs now say]():

```
Updating searchtools for Read the Docs search... build succeeded, 29 warnings.
```

I am trying to address those though.

Reply:
There are two remaining warnings:

- `WARNING: Duplicate explicit target name: "1".` because now I'm telling `sphinx-autoapi` to show the inherited properties, and this results in several `.. [1] https://en.wikipedia.org/wiki/Great-circle_distance` on the same page because they're inherited from Astropy.
- `/home/docs/checkouts/readthedocs.org/user_builds/poliastro/checkouts/1531/docs/source/autoapi/poliastro/twobody/propagation/_compat/index.rst:32: WARNING: more than one target found for cross-reference '__class__': poliastro.twobody.propagation.cowell.__class__, poliastro.twobody.propagation.danby.__class__, poliastro.twobody.propagation.farnocchia.__class__, poliastro.twobody.propagation.gooding.__class__, poliastro.twobody.propagation.markley.__class__, poliastro.twobody.propagation.mikkola.__class__, poliastro.twobody.propagation.pimienta.__class__, poliastro.twobody.propagation.recseries.__class__, poliastro.twobody.propagation.vallado.__class__`. I don't understand why this happens, because supposedly the module is private.

Reply:
Now only the first issue remains, but I don't see an easy way out.